aws/data_source_aws_ssm_document_test.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSSsmDocumentDataSource_basic(t *testing.T) {
+	resourceName := "data.aws_ssm_document.test"
+	name := "test_document"
```

Comment:
Nit: We should randomize the naming to prevent testing collisions and enable future parallel testing.